www/src/components/CarbonAds.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'react';
+
+const CarbonAds = () => {
+  const ref = useRef();
+
+  useEffect(() => {
+    if (typeof document !== 'undefined') {
+      const script = document.createElement('script');
+      script.async = true;
+      script.id = '_carbonads_js';
+      script.type = 'text/javascript';
+      script.src =
+        '//cdn.carbonads.com/carbon.js?serve=CE7IP2QY&placement=react-bootstrapgithubio';
+
+      ref.current.appendChild(script);
```

Comment:
Does this need any clean up logic?

Reply:
Don't think we need any.  I think the node unmounting should be enough

Reply:
Added one just in case 😄